src/main/java/alison/command/AddCommand.java

```diff
@@ -0,0 +1,32 @@
+package alison.command;
+
+import alison.exception.AlisonException;
+import alison.task.Deadline;
+import alison.task.Task;
+import alison.tool.Storage;
+import alison.tool.TaskList;
+import alison.tool.Ui;
+
+public class AddCommand extends Command {
+
+    private Task task;
+
+    public AddCommand(Task added) {
+        this.task = added;
```

Comment:
The Java coding standard does not seem to specify if "this" is recommended or not recommended so both is probably good. Personally, I also prefer to use "this" in my constructors because I like to know what fields I'm setting!

Reply:
I personally prefer to use "this" keyword to make it clear that I am setting/using an attribute in the class.